crates/pixi_manifest/src/manifest.rs

```diff
@@ -2142,4 +2143,22 @@ bar = "*"
             Caused by: Found invalid characters for git revision 'main', branches and tags are not supported yet
         "###);
     }
+
+    #[test]
+    fn test_docs_pixi_manifests() {
+        let location = "../../docs/source_files/pixi_tomls/*.toml";
+
+        // Check that the glob is not empty
+        assert!(glob(location).unwrap().count() > 0);
+
+        for entry in glob(location).unwrap() {
```

Comment:
```suggestion
        let entries = glob(location).unwrap();
        assert!(entries.count() > 0);

        for entry in entries {
```

Reply:
I've been fighting with this one test for over 30 minutes.

The issues is that count() consumes entries. And i'm not allowed to clone() it. 

So this was the solutions.

But my real hope is that we could use #[files()] in combination with rstest but I wasn't able to get it working.

Reply:
I see, then let's leave it as is :)